### DISCUSSION TITLE
Regional extension pricing (PPP)

### DISCUSSION THREAD

**Discussion by xbc5** [Glench/ExtPay#236] [Ideas]
Developers that charge a flat rate across different economies make it unfair to smaller, emerging economies, and is likely leading to missed opportunities IMHO.

I wonder if you would consider translating costs via [PPP](https://en.wikipedia.org/wiki/Purchasing_power_parity)? Essentially it describes what consumers can afford to spend relative to their local economies. For example:
- flat conversion: 20 GBP => ~653k VND  -- this is a straight currency convesion
- PPP adjusted: 20 GBP => ~213k VND -- this considers other economic factors

If I were to think that £2 was a fair price for a coffee in Vietnam, I wouldn't charge 65k VND, I'd charge 21k VND.

Here's [information on the calculation](https://www.worldbank.org/en/programs/icp/brief/methodology-calculation). It requires some digging to find complete, and up-to-date data resource but there's the [OECD price level index](https://www.oecd.org/en/data/indicators/price-level-indices.html?oecdcontrol-00b22b2429-var3=2022) (incomplete), and the [World Bank Data Catalogue](https://datacatalog.worldbank.org) (huge resource) to get you started.

Some informally call it the [Big Max Index](https://www.economist.com/big-mac-index), IPad index etc. These are slightly different calculations, and they're weighted by import and trade costs -- which might not be the most effective analogue for a straight digital transaction.

What do you guys think?

**Comment by Glench** [comment]
I like this idea and have thought about it myself. Are there any off-the-shelf indexes to calculate this information? I was thinking about just using Apple's conversion rate tables in app stores in different economies.
